### DISCUSSION TITLE
Speech/Audio/Video to Text (Transcriptions)

### DISCUSSION THREAD

**Discussion by eleazar7** [immich-app/immich#3629] [General]
Speech/Audio/Video to Text (Transcriptions)
could this be added? this would add to the search!
You could now search based on what is said in any Audio/Video with Transcriptions! 
or add a plugin or extension of some kind 

Whisper would be great!
https://github.com/openai/whisper

**Comment by bo0tzz** [comment]
(conceptually) related: #3168

There have been some ideas floated about building some sort of plugin system in the (far) future. I think something like this would be a great fit for that.

**Reply by eleazar7** [reply]
yeah, that would be cool. I believe this would be used a lot!

**Comment by Chuckame** [comment]
Now that the OCR feature is here (v2.2.0), maybe speech-to-text is now possible!

**Comment by ChickenSaysBak** [comment]
I would love to see this added!
